src/contexts/UserContext.js

```diff
@@ -1,46 +1,44 @@
-import React, { createContext, useState, useMemo, useCallback } from 'react';
+import React, { createContext, useState, useMemo, useCallback } from "react";
 
-import { UserClient } from '../services/User';
+import { UserClient } from "../services/User";
```

Comment:
use absolute imports `services/User`

Reply:
There are more imports like this one. We should do a project-wide cleanup